buildSrc/src/main/java/com/r3/csde/ProjectUtils.java

```diff
@@ -29,36 +29,7 @@ void rpcWait(int millis) {
         }
     }
 
-    void rpcWait() {
-        rpcWait( pc.retryWaitMs);
-    }
-
-    public LinkedList<String> getConfigX500Ids(String configFile) throws IOException {
-        LinkedList<String> x500Ids = new LinkedList<>();
-//        com.fasterxml.jackson.databind.ObjectMapper mapper = new com.fasterxml.jackson.databind.ObjectMapper();
-        ObjectMapper mapper = new ObjectMapper();
-
-
-        FileInputStream in = new FileInputStream(configFile);
-        com.fasterxml.jackson.databind.JsonNode jsonNode = mapper.readTree(in);
-        for( com.fasterxml.jackson.databind.JsonNode identity:  jsonNode.get("identities")) {
-            x500Ids.add(jsonNodeToString(identity));
-        }
-        return x500Ids;
-    }
-
-    public String jsonNodeToString(com.fasterxml.jackson.databind.JsonNode jsonNode) {
-        String jsonString = jsonNode.toString();
-        return jsonString.substring(1, jsonString.length()-1);
-    }
-
-    public void downloadFile(String url, String targetPath) {
-        Unirest.get(url)
-                .asFile(targetPath)
-                .getBody();
-    }
-
-    public void reportError(@NotNull HttpResponse<JsonNode> response) throws CsdeException {
+    public void reportError(HttpResponse<JsonNode> response) throws CsdeException {
```

Comment:
isn't having `@NotNull` better for stricter code and to fail earlier ?

Reply:
I ned to look at this, I was under the impression that @Notnull was a jetbrains library, I didn't want to make the assumption that the developer was using IntelliJ.

Reply:
Confirmed, this is from a jet brains library. I believe it is the only place we import a jet brains library in the code base. 

The function itself is only a pretty print for a response object, and will be refactored out when this code is migrated to the plugin. (This throwing a null exception here is the least of the problems with the code, it currently has insufficient exception handling. )

It seems a poor reason to bring an extra dependency in with the overhead of dependency checking. So I propose leaving it out.